datasets/pg19/README.md

```diff
@@ -1,6 +1,7 @@
 ---
 language:
 - en
+license: apache-2.0
```

Comment:
The license tag should be an array.

Reply:
hmm that's not what's documented on https://huggingface.co/docs/hub/datasets-cards =) 

<img width="1075" alt="Screenshot 2022-08-11 at 16 12 20" src="https://user-images.githubusercontent.com/326577/184153709-02b0779a-aae2-4202-929c-8d940949d5c9.png">

Reply:
but ok i'll push this for consistency

Reply:
Good catch: we should align docs and validation implementation:

https://github.com/huggingface/datasets/blob/cfae8ac7c896772af537441508aaae2391aea53a/src/datasets/utils/metadata.py#L232